deploy/kubernetes/helm-charts/altinn-runtime/templates/service.yaml

```diff
@@ -6,10 +6,8 @@ metadata:
     run: altinn-runtime
 spec:
   ports:
-  {{- range $port := .Values.service.ports }}
-    - port: {{ $port.port }}
-      targetPort:  {{ $port.targetPort }}
-      protocol:  {{ $port.protocol }}
-  {{- end }}
+    - port: {{ .Values.image.containerPort }}
+      targetPort:  {{ .Values.image.containerPort }}
+      protocol:  TCP
```

Comment:
A deployment/pod might map to several ports, this limits us to only using one port per deployment/pod.

Reply:
Keeping template simple until we need the feature.